frontend/src/components/Block/Block.tsx

```diff
@@ -258,27 +256,27 @@ const Block = () => {
             // Block views
             // -------------------------
             case "TRIAL_VIEW":
-                return <Trial {...attrs} />;
+                return <Trial key={key} {...attrs} />;
```

Comment:
You might as well add the key to the `const attrs = { ... };` part above, or not?

Reply:
Nope, that's going to reintroduce the warning we're trying to get rid of (#1221)